swagger.yaml

```diff
@@ -0,0 +1,96 @@
+openapi: 3.0.3
+info:
+  title: Agartex Service
+  description: |-
+    This is a server for compiling Latex documents and authenticating users
+  version: 0.0.1
+servers:
+  - url: http://localhost:3000
+  - url: some_domain
+tags:
+  - name: user
+    description: Operations about user
+paths:
+  /user:
+    post:
+      tags:
+        - user
+      summary: Register user
+      description: This will register a new user with the given email and password.
+      operationId: createUser
+      requestBody:
+        description: Created user object
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/User'
+      responses:
+        200:
+          description: successful operation
+        400:
+          description: validation error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Message'
+  /user/login:
```

Comment:
here idk what is the convention but we are creating a user session, so MAYBE `POST /users/sessions`?

Reply:
after reconsideration, i think `POST /user-sessions/` is better, a session is a resource parallel to user, not under it (though they are linked)

Reply:
I'll just make it `POST /sessions`.